Wheels not moving on kinetic-devel branch

Hi folks!

I'm trying to use the kinetic-devel branch instead of the indigo-devel that is distributed with apt for kinetic for my Magni.
It works fine with indigo-devel, but when I switch to the new branch I get this error:
```
[ERROR] [1625944236.182432997]: Failed to read from I2C device /dev/i2c-1!  ERROR: Remote I/O error
[ERROR] [1625944236.183686735]: Error -2 in reading MCB option switch at 8bit Addr 0x40
```
The wheels don't spin when I publish to /cmd_vel and 
`rostopic echo /motor_power_active` gives me `data: False`

I thought it could be something with the new `system_control` topic, but publishing `motor_control enable`  and `speed_control enable ` did not seem to help.

I have this firmware version at the moment:
```
fw revision  40
fw daycode   20201209
hw revision  4.9
```
Any idéas about why this is happening? Maybe I have old hardware that is not compatible? Do I need to do something to enable the motors when using the new branch?

--- a/include/ubiquity_motor/motor_hardware.h
+++ b/include/ubiquity_motor/motor_hardware.h
@@ -50,6 +50,9 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <gtest/gtest_prod.h>
 
+// Mininum hardware versions required for various features
+#define MIN_HW_OPTION_SWITCH 50
+
 struct MotorDiagnostics {
     MotorDiagnostics()
         : odom_update_status(

--- a/src/motor_hardware.cc
+++ b/src/motor_hardware.cc
@@ -480,6 +480,7 @@ void MotorHardware::requestSystemEvents() {
 // The 1st firmware version this is set for is 32, before it was always 1
 void MotorHardware::setHardwareVersion(int32_t hardware_version) {
     ROS_INFO("setting hardware_version to %x", (int)hardware_version);
+    this->hardware_version = hardware_version;
     MotorMessage mm;
     mm.setRegister(MotorMessage::REG_HARDWARE_VERSION);
     mm.setType(MotorMessage::TYPE_WRITE);

--- a/src/motor_node.cc
+++ b/src/motor_node.cc
@@ -191,7 +191,7 @@ void initMcbParameters(std::unique_ptr<MotorHardware> &robot )
 
     // Tell the MCB board what the I2C port on it is set to (mcb cannot read it's own switchs!)
     // We could re-read periodically but perhaps only every 5-10 sec but should do it from main loop
-    if (robot->firmware_version >= MIN_FW_OPTION_SWITCH) {
+    if (robot->firmware_version >= MIN_FW_OPTION_SWITCH && robot->hardware_version >= MIN_HW_OPTION_SWITCH) {
         g_firmware_params.option_switch = robot->getOptionSwitch();
         ROS_INFO("Setting firmware option register to 0x%x.", g_firmware_params.option_switch);
         robot->setOptionSwitchReg(g_firmware_params.option_switch);

